### DISCUSSION TITLE
Does placeholder work for images in cards?

### DISCUSSION THREAD

**Discussion by dosstx** [twbs/bootstrap#34682] [General]
I noticed that the placeholder still requires an actual image source, correct?

I thought the point of placeholder was to have some kind of "filler" ?

`<img src="..." class="card-img-top" alt="...">`

How does one use a placeholder image here without referencing an actual image resource?

**Comment by ghost** [comment]
Simple
Use css.

.placeholder {
filter:brightness(5%) contrast(5%);
}

My work on it
https://sahib111.github.io/Diary-Blog/



See in desktop version to see the animation.


As I see now, you want lazy loading script?

Try setting image using css as background:url("") and let me know.
